src/token/token.resolver.ts

```diff
@@ -0,0 +1,58 @@
+import { Args, Int, Mutation, Resolver } from '@nestjs/graphql'
+import { Prisma } from '@prisma/client'
+import { Roles } from '../auth/decorators'
+import { Token } from '../token/entities/token.entity'
+import { TokenService } from '../token/token.service'
+
+@Resolver(() => Token)
+export class TokenResolver {
```

Comment:
You should to Add `getAllTokens` with pagination ability. (For example, watch News module)

Reply:
i did it